apps/networkmonitor/networkmonitor.nim

```diff
@@ -519,8 +520,31 @@ when isMainModule:
   waitFor node.mountRelay()
   waitFor node.mountLibp2pPing()
 
-  # Subscribe the node to the default pubsubtopic, to count messages
-  subscribeAndHandleMessages(node, DefaultPubsubTopic, msgPerContentTopic)
+  if conf.rlnRelayEthContractAddress != "":
+
+    let rlnConf = WakuRlnConfig(
+      rlnRelayDynamic: conf.rlnRelayDynamic,
+      rlnRelayCredIndex: some(uint(0)),
+      rlnRelayEthContractAddress: conf.rlnRelayEthContractAddress,
+      rlnRelayEthClientAddress: conf.rlnRelayEthClientAddress,
+      rlnRelayCredPath: "",
+      rlnRelayCredPassword: "",
+      rlnRelayTreePath: conf.rlnRelayTreePath,
+    )
+
+    try:
+      waitFor node.mountRlnRelay(rlnConf)
+    except CatchableError:
+      error "failed to setup RLN", err=getCurrentExceptionMsg()
+      quit 1
+
+  node.mountMetadata(1).isOkOr:
+    error "failed to mount waku metadata protocol: ", err=error
+    quit 1
+
+  for pubsubTopic in conf.pubsubTopics:
```

Comment:
in case you want to simplify the cli config just added this:
https://github.com/waku-org/nwaku/blob/master/apps/wakunode2/networks_config.nim#L20-L40

Reply:
Ok, I just imported the config from `wakunode2` - I don't like it, but it's a reasonable start, if this gets moved to more general location, we can change this. But I felt like this is a better approach than to copy it to networkmonitor app